### PR TITLE
Add swagger ui 3.x

### PR DESCRIPTION
**Pull request by ddanielroche**
I think that @Sensetivity work is very good and I would like to see these changes.
I added support for the property urls

### CODE REVIEW

File: src/SwaggerUIAssetOverrides.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+class SwaggerUIAssetOverrides extends AssetBundle
+{
+    public $sourcePath = '@vendor/light/yii2-swagger/src';
+
+    public $js = [
+    ];
+
+    public $css = [
+        'swagger-overrides.css',
```
Comment (lichunqiang) [inline src/SwaggerUIAssetOverrides.php]: The css file is not exists?

Comment (ddanielroche) [reply src/SwaggerUIAssetOverrides.php]: Sorry, I forgot to remove SwaggerUIAssetOverrides.php

**Comment by ddanielroche** [comment]
Removing SwaggerUIAssetOverrides, removing unused imports.

**Comment by lichunqiang** [comment]
54fe1c97a27178cca954d62b2a7a8f3ad02b3ad5 merged

**Comment by ddanielroche** [comment]
I have tried the changes made by you and I find them very good. When you release version 2.0. I am proud to be able to collaborate in the project
Have a nice day.

**Pull request closed without merging**